server/aesdsocket.c

```diff
@@ -14,19 +16,32 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <syslog.h>
+#include <time.h>
 #include <unistd.h>
 
 #define PORT "9000"
 #define DATA_FILE "/var/tmp/aesdsocketdata"
 #define BUF_SIZE 1024
+#define BUFF_SIZE 1024
 
 int socket_fd = 0;
 int client_socket_fd = 0;
 int file_fd = 0;
 int transfer_exit = 0;
+int status = 0;
```

Comment:
@aneesh1298 I don't see that this is used but you'd want to be careful about using globals with threads.  If it does need to be global you should probably give it a better name.